dagger.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dagger-dev",
-  "engineVersion": "v0.15.2",
+  "engineVersion": "v0.15.3",
```

Comment:
Ahh, I would not bump this right now, that's being done in https://github.com/dagger/dagger/pull/9474, and also requires the runners to be updated (which is what's making CI go red on this PR right now).

Reply:
honestly don't know why i did this, removed